controllers/keda/scaledobject_controller.go

```diff
@@ -62,6 +62,7 @@ import (
 // +kubebuilder:rbac:groups="*",resources="*",verbs=get
 // +kubebuilder:rbac:groups="apps",resources=deployments;statefulsets,verbs=list;watch
 // +kubebuilder:rbac:groups="coordination.k8s.io",namespace=keda,resources=leases,verbs="*"
+// +kubebuilder:rbac:groups="",resources="limitranges",verbs=list;watch
```

Comment:
do we need `get` verb? (maybe we don't)

Reply:
Yeah, for now, we are only listing the `LimitRange` objects. So, excluded `get`.